docs/tools/config_system.md

```diff
@@ -91,87 +217,31 @@ In this JSON file:
-
-If the source tree has code for more than one library, each library needs its own `mbed_lib.json` file in its root folder.
-
-##### Configuration data in targets
-
-Like libraries, targets can define their own configuration data. Additionally, targets can override the configuration of the target(s) they inherit from (for more details about how do define a target and target inheritance, check [our configuring targets documentation](/docs/development/tools/adding-and-configuring-targets.html)). Target configuration data is defined in `targets.json` using `config`, as described in the section about [defining configuration parameters](#defining-configuration-parameters). An example for a hypothetical `Base` target is given below:
-
-```
-"Base": {
-    "core": "Cortex-M0",
-    "extra_labels": ["BASE_LABEL"],
-    "config": {
-        "serial_console_speed": {
-            "help": "Baud rate of the serial console",
-            "value": 115200,
-            "macro_name": "MBED_SERIAL_UART_SPEED"
-        },
-        "stack_size": {
-            "help": "Initial stack size of the application",
-            "value": 128
-        }
-    }
-}
-```
-
-Like library parameters, target-defined parameters have an implicit prefix. For a target, the prefix is always called `target` (no matter what the actual target name is), so the above configuration parameters will be accessible outside the definition in `Base` (and any other target) as `target.serial_console_speed` and `target.stack_size`.
-
-Targets can inherit from other targets, and their configuration data is also inherited. A target that inherits from one or more other targets can add new parameters in its own `config` section and can also override the configuration parameters defined by its parent(s) in the `overrides` section. For example:
-
-```
-"Derived": {
-    "inherits": ["Base"],
-    "extra_labels_add": ["NXP"],
-    "config": {
-        "my_own_config": {
-            "help": "My very own configuration parameter",
-            "value": 0
-        }
-    },
-    "overrides": {
-        "stack_size": 256
-    }
-}
-```
-
-`Derived` above defines its own configuration parameter called `my_own_config` and inherits the configuration parameters from `Base`, so its configuration parameters are `serial_console_speed`, `stack_size` and `my_own_config`. It also overrides the value of the `stack_size` parameter defined in `Base`. This means that:
+All configuration parameters defined in `mylib` have a `mylib.` prefix. Outside `mylib`, `buffer_size` is accessible using the name `mylib.buffer_size`. An application will be able to override the value of this parameter, as described in [this section](#configuration-data-in-applications).
```

Comment:
Outside myLib in another mbed_lib.json or mbed_app.json. 

'this section' probably gives the answer but two or three more words could allow me to continue reading without clicking.

Reply:
Ah. I caused some confusion. I'll correct this section.